internal/xds/translator/testdata/out/xds-ir/http-route-zonal-routing.endpoints.yaml

```diff
@@ -0,0 +1,125 @@
+- clusterName: first-route-dest/backend/0
+  endpoints:
+  - lbEndpoints:
+    - endpoint:
+        address:
+          socketAddress:
+            address: 1.1.1.1
+            portValue: 50001
+      loadBalancingWeight: 1
+    - endpoint:
+        address:
+          socketAddress:
+            address: 1.1.1.4
+            portValue: 50001
+      loadBalancingWeight: 1
+    locality:
+      zone: us-west-2a
+  - lbEndpoints:
+    - endpoint:
+        address:
+          socketAddress:
+            address: 1.1.1.2
+            portValue: 50001
+      loadBalancingWeight: 1
+    - endpoint:
+        address:
+          socketAddress:
+            address: 1.1.1.5
+            portValue: 50001
+      loadBalancingWeight: 1
+    locality:
+      zone: us-west-2b
+  - lbEndpoints:
+    - endpoint:
+        address:
+          socketAddress:
+            address: 1.1.1.3
+            portValue: 50001
+      loadBalancingWeight: 1
+    - endpoint:
+        address:
+          socketAddress:
+            address: 1.1.1.6
+            portValue: 50001
+      loadBalancingWeight: 1
+    locality:
+      zone: us-west-2c
+- clusterName: first-route-dest/backend/1
+  endpoints:
+  - lbEndpoints:
+    - endpoint:
+        address:
+          socketAddress:
+            address: 2.2.2.2
+            portValue: 50002
+      loadBalancingWeight: 1
+    - endpoint:
+        address:
+          socketAddress:
+            address: 2.2.2.3
+            portValue: 50002
+      loadBalancingWeight: 1
+    - endpoint:
+        address:
+          socketAddress:
+            address: 2.2.2.4
+            portValue: 50002
+      loadBalancingWeight: 1
+    - endpoint:
+        address:
+          socketAddress:
+            address: 2.2.2.5
+            portValue: 50002
+      loadBalancingWeight: 1
+    - endpoint:
+        address:
+          socketAddress:
+            address: 2.2.2.6
+            portValue: 50002
+      loadBalancingWeight: 1
+    loadBalancingWeight: 10
```

Comment:
shouldnt these be `1`, since we are moving the load balancing weights to the route level

Reply:
resolving this, this is a no-op, there's one locality per cluster